tests/test-ucx-w-dask-workers.py

```diff
@@ -0,0 +1,109 @@
+import os
+import pytest
+from distributed.utils import format_bytes
+import numpy as np
+import asyncio
+import dask.dataframe as dd
+from distributed import Scheduler, Worker, Client, Nanny, wait
+from distributed.utils import log_errors
+
+from cudf.tests.utils import assert_eq
+
+#async with Nanny(s.address, protocol='ucx', nthreads=1,
+# nanny is really a worker running on a defined CUDA DEVICE
+protocol = 'ucx'
+interface = 'ib0'  # Ff changing CUDA_VISIBLE_DEVICES CHECK IB Controller
+
+w_env = {
+    "UCX_RNDV_SCHEME": "put_zcopy",
+    "UCX_MEMTYPE_CACHE": "n",
+    "UCX_TLS": "rc,cuda_copy,cuda_ipc",
+    "CUDA_VISIBLE_DEVICES": "0,1",
+}
+os.environ.update(w_env)
+
+w_env_2 = w_env.copy()
+w_env_2["CUDA_VISIBLE_DEVICES"] = "1,0"
+
+async def f(cudf_obj_generator):
+    async with Scheduler(protocol=protocol, interface=interface,
+    dashboard_address=':8789') as s:
+        async with Nanny(s.address, protocol=protocol, nthreads=1,
+                memory_limit='32GB', interface=interface,
+                env=w_env,
+                ) as w:
+            async with Nanny(s.address, protocol=protocol,memory_limit='32gb',
+                    env=w_env_2, interface=interface,
+                    nthreads=1) as w2:
+                async with Client(s.address, asynchronous=True) as c:
+                    with log_errors():
+
+                        def set_nb_context(x=None):
+                            import numba.cuda
+                            try:
+                                numba.cuda.current_context()
+                            except Exception:
+                                print("FAILED EXCEPTION!")
```

Comment:
I think that we should avoid print statements in testing code.  Lets assume that no one will be reading these logs.  If we want to check something, we should check it programmatically.